myhpom/urls.py

```diff
@@ -29,10 +29,14 @@
         name='verify_account'),
 
     url(r'^download/(?P<path>.*)$', myhpom_views.irods_download, name='irods_download'),
+
+    url(r'^document/cloudfactory_response/?$', 
+        myhpom_views.cloudfactory_response, name='cloudfactory_response'),
```

Comment:
flake8 complaining about whitespace in here. Sigh, we should really just add flake8 to our CI pipeline :/

Reply:
Yes, but we should tone down some of the warnings a bit IMO – like whitespace at the end of a docstring or comment line.